src/funnels/automation/ConduitMover.sol

```diff
@@ -99,7 +99,9 @@ contract ConduitMover {
         unchecked { configs[from][to][gem].num = cfg.num - 1; }
         configs[from][to][gem].zzz = uint32(block.timestamp);
 
-        if (from != buffer) ConduitLike(from).withdraw(ilk, gem, cfg.lot);
+        if (from != buffer) {
+            require(ConduitLike(from).withdraw(ilk, gem, cfg.lot) == cfg.lot, "ConduitMover/lot-withdraw-failed");
+        }
         if (to   != buffer) ConduitLike(to).deposit(ilk, gem, cfg.lot);
```

Comment:
Maybe I'd use the {} now here as well, and not sure if the extra spacing for alignment still makes sense now.

Reply:
Done.